storybook/storybook-react/src/MegaMenu/MegaMenu.stories.tsx

```diff
@@ -156,147 +76,79 @@ export const MultipleCategories: Story = {
       <Grid key="nav" paddingVertical="medium">
         <Grid.Cell span={{ narrow: 4, medium: 8, wide: 8 }}>
           <Heading level={3} size="level-3">
-            Thema&rsquo;s
+            Thema’s
           </Heading>
           <MegaMenu.ListCategory>
-            <UnorderedList markers={false}>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Afval
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Bestuur en organisatie
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Bouw- en verkeersprojecten
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Burgerzaken
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Diversiteit
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Gemeentebelastingen
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Gezondheidsdienst (GGD)
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Kunst en cultuur
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Ondernemen
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Onderwijs en jeugd
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Parkeren
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Sport
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Stadsdelen
-                </Link>
-              </UnorderedList.Item>
-              <UnorderedList.Item>
-                <Link variant="inList" href="#">
-                  Subsidies
-                </Link>
-              </UnorderedList.Item>
-            </UnorderedList>
+            <LinkList>
+              <LinkList.Link href="#" size="large">
```

Comment:
Out of scope for this PR, but I missed it in the LinkList PR: do we really want a `large` link variant? It looks a little off imo

Reply:
Noted for discussion.